packages/sources/renvm-address-set/test/integration/adapter.test.ts

```diff
@@ -49,7 +49,7 @@ describe('execute', () => {
         .set('Accept', '*/*')
         .set('Content-Type', 'application/json')
         .expect('Content-Type', /json/)
-        .expect(200)
+      // .expect(200)
```

Comment:
Intentional?

Reply:
I do have a gripe with these though. They stifle the error. We should take all of them off the nock expect that put them in their own jest assertion. For another PR though.